substrate/frame/staking/src/ledger.rs

```diff
@@ -32,22 +32,26 @@
 //! state consistency.
 
 use frame_support::{
-	defensive,
+	defensive, ensure,
 	traits::{LockableCurrency, WithdrawReasons},
 };
-use sp_staking::StakingAccount;
+use sp_staking::{OnStakingUpdate, Stake, StakerStatus, StakingAccount, StakingInterface};
 use sp_std::prelude::*;
 
 use crate::{
 	BalanceOf, Bonded, Config, Error, Ledger, Payee, RewardDestination, StakingLedger, STAKING_ID,
 };
 
-#[cfg(any(feature = "runtime-benchmarks", test))]
-use sp_runtime::traits::Zero;
+impl<T: Config> Into<Stake<BalanceOf<T>>> for StakingLedger<T> {
```

Comment:
it is generally more idiomatic in Rust to implement `From<X> for Y` than `Into<Y> for X` (although truthfully I sometimes find the latter more intuitive).

Reply:
I agree. In this case, the another reason to implement `Into` instead is that we're implementing the conversion from a a higher level type (`StakingLedger`) into a lower level/primitive type (`Stake`).

Reply:
@georgepisaltu I like that, thanks for the suggestion. Changed the code accordingly.